CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1.1.0 (11/12/2019)
+
+**Common**
+* The deprecated `cancel` method in `CancellationToken` removed.
+* The curl network implementation fixed to compile on 32 bits architecture.
+* The `disk_path` field in `CacheSettings` is deprecated. The `disk_path_mutable` field should be used instead.
+* A new synchronous `CallApi` method was added to `OlpClient`.
+* A bug is fixed when cmake was not able to check that the `pipe2` symbol exists.
+
+**olp-cpp-sdk-authentication**
+* The `SignInClient` method in `AuthenticationClient` is deprecated in favor of the newly introduced `SignInClient` method with a different signature.
+* A `scope` support was added through the `SignInProperties` struct.
+* A new `error_id` field was added to `ErrorResponse` structure. It used to propagate `ERROR_ID` string from the authentication error response.
```

Comment:
```suggestion
* `error_id` field now added to `ErrorResponse` structure. It will be used to propagate the `errorId` string from the authentication error response.
```

Reply:
```suggestion
* The `error_id` field was added to the `ErrorResponse` structure. You can use it to get the `errorId` string from the authentication error response.
```